Dockerfile

```diff
@@ -12,6 +12,5 @@ ENV USER_UID=10001
 # or the codes will use it and cause an error, the container cannot be started.
 COPY --from=builder /go/src/open-cluster-management.io/placement/placement /
 COPY --from=builder /go/src/open-cluster-management.io/placement/e2e.test /
-RUN microdnf update && microdnf clean all
```

Comment:
```
(microdnf:13): librhsm-WARNING **: 07:28:38.569: Found 0 entitlement certificates
Downloading metadata...
error: cannot update repo 'ubi-8-baseos': Cannot download repomd.xml: Cannot download repodata/repomd.xml: All mirrors were tried; Last error: Curl error (6): Couldn't resolve host name for https://cdn-ubi.redhat.com/content/public/ubi/dist/ubi8/8/x86_64/baseos/os/repodata/repomd.xml [Could not resolve host: cdn-ubi.redhat.com]
running 'microdnf update && microdnf clean all' failed with exit code 1
make: *** [image-placement] Error 1
```

because this line will fail in our build environment. looks like it's trying to connect an internal repository.

Reply:
I guess other repos may have the same issue: 
https://github.com/open-cluster-management-io/managed-serviceaccount/blob/441d5a67f6d12192d3a576190e76c8ae360da446/Dockerfile#L13

Reply:
@xuezhaojun thx for reminding, we didnt cover the addon repos in our downstream build yet. will a PR remove the line too.